src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -42,15 +53,11 @@ class Crystal::Doc::Generator
       filename = "Readme.md"
     end
 
-    if filename
-      body = File.read(filename)
-    else
-      body = ""
-    end
-
-    body = String.build do |io|
-      Markdown.parse body, MarkdownDocRenderer.new(program_type, io)
-    end
+    body = if filename
```

Comment:
I much prefer the old style of having `body =` inside the block. Also isn't this `String.build` redundant? Doesn't `doc` return a string anyway?